src/index.css

```diff
@@ -1,13 +1,22 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
 body {
```

Comment:
Can we add some general padding so things aren't squeezed right up to the edge of the window?